packages/rx_bloc_cli/mason_templates/bricks/feature_login/__brick__/lib/feature_login/ui_components/sign_up_button.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+
+import '../../app_extensions.dart';
+import '../../base/extensions/async_snapshot_extensions.dart';
+import '../blocs/login_bloc.dart';
+
+class SignUpButton extends StatelessWidget {
+  const SignUpButton({super.key});
+
+  @override
+  Widget build(BuildContext context) => RxBlocBuilder<LoginBlocType, bool>(
```

Comment:
Please make sure all ui_components are decoupled from the state management.

Reply:
Fixed!